lib/zendesk_apps_support/installed.rb

```diff
@@ -12,6 +12,15 @@ def initialize(appsjs, installations = [])
       @installations = installations
     end
 
+    def obj(options = {})
```

Comment:
This is called on `fetch` for `/apps/installed.json` here https://github.com/zendesk/zendesk_app_market/pull/3911/files#diff-637b58a4e4ebc7da13a8c8c400fff49aR60

So maybe this should be `get_app_installation_metadata`?